src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1044,18 +1044,19 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
 
                         if (Logging.Analytics.ReportingAnalytics)
                         {
-                            var modifiedNodes = "";
                             if (updateTask.ModifiedNodes != null && updateTask.ModifiedNodes.Any())
                             {
-                                modifiedNodes = updateTask.ModifiedNodes
-                                    .Select(n => n.GetOriginalName())
-                                    .Aggregate((x, y) => string.Format("{0}, {1}", x, y));
+                                // Send analytics for each of modified nodes so they are counted individually
+                                foreach (var node in updateTask.ModifiedNodes)
+                                {
+                                    // Starting from Dynamo 2.11.0, tracking node execution as generic event so
+                                    // it is distinguished with the previous aggregated performance event
```

Comment:
Regarding this comment, I would abstain from including `Starting from Dynamo 2.11.0,`. Since we are using git it's easy to backtrack changes without having to add timestamps or version marks.